bot/src/commands/revert-last-command.js

```diff
@@ -0,0 +1,54 @@
+import { get, post } from '../utils';
+import Commands from './index';
+
+function getCommandByName(bot, name) {
+  for (let Command of bot.commands) {
+    if (Command.name === name) {
+      return Command;
+    }
+  }
+  return null;
+}
+
+export const RevertLastCommand = {
+  name: 'RevertLastCommand',
+
+  matches(str) {
+    return str === 'undo';
+  },
+
+  execute(bot, author, channel, text) {
+    const log = bot.getActionLog(author, channel);
+    if (log.length === 0) {
+      return Promise.resolve('There is no command to revert');
+    }
```

Comment:
A nit here and in a few other places:  please insert a blank line after all terminating `if` blocks.